style.css

```diff
@@ -382,3 +417,173 @@ body {
   color: white;
   cursor: pointer;
 }
+
+.cont-menu-list {
+  display: none;
+}
+
+@media (min-width: 600px) {
+  .cont-menu-list {
+    list-style: none;
+    display: flex;
+  }
+
+  .menu__cont i {
+    display: none;
+  }
+
+  .link-menu {
+    text-decoration: none;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: black;
+  }
+
+  .menu-list > * {
+    margin-right: 1rem;
+  }
+
+  .intro {
+    width: 80%;
+    margin: 7rem auto 0 auto;
+  }
+
+  .container__card {
+    width: 75%;
+  }
+
+  .cont-inp-form {
+    width: 70%;
+    margin: 0 auto;
+  }
+
+  .item-field {
+    width: 90%;
+    margin-right: auto;
+    margin-left: auto;
+  }
+
+  .cont-btn-form {
+    align-self: center;
+  }
+}
+
+@media (min-width: 900px) {
```

Comment:
Please write the value for min-width property should in range of 992px - 1100px.